### PR TITLE
Docs: Updated Config providers docs

### PR DESCRIPTION
**Pull request by dat-a-man**
### Description
Documentation update

### Related Issues

- Fixes #...
- Closes #...
- Resolves #...

### CODE REVIEW

File: docs/website/docs/general-usage/credentials/config_providers.md [hunk 1]
```diff
@@ -92,12 +92,26 @@ sections and key names are separated by "." into the provider-specific formats.
...
+   [sources.github]
+   access_token = "GITHUB_API_TOKEN"
+   ```
+   Use the following environment variable:
+   ```bash
+   export SOURCES__GITHUB__ACCESS_TOKEN="your_token_here"
+   ```
+
+
+1. When `dlt` evaluates a request like  'dlt.secrets["my_section.gcp_credentials"]', it looks for it as follows:
```
Comment (AstrakhantsevaAA) [inline docs/website/docs/general-usage/credentials/config_providers.md:108]: replace this sentence with this please:
When `dlt` evaluates the request `dlt.secrets["my_section.gcp_credentials"]` it must find the `private_key` for Google credentials.  It looks for it as follows:

Comment (dat-a-man) [reply docs/website/docs/general-usage/credentials/config_providers.md:108]: done

**Review by AstrakhantsevaAA — CHANGES_REQUESTED** [review]
fragments of code should be highlighted as code
dlt.secrets[“my_section.gcp_credentials”] -> `dlt.secrets["my_section.gcp_credentials"]`

**Comment by dat-a-man** [comment]
yeah, done.

**Review by AstrakhantsevaAA — CHANGES_REQUESTED** [review]
private_key should be as code too

**Review by AstrakhantsevaAA — APPROVED** [review]

**Review by AstrakhantsevaAA — APPROVED** [review]

**Pull request merged**